cookbooks/bcpc-hadoop/templates/default/hdp_container-executor.cfg.erb

```diff
@@ -2,4 +2,4 @@ yarn.nodemanager.local-dirs=<%=@mounts.map{ |d| "file:///disk/#{d}/yarn/local" }
 yarn.nodemanager.linux-container-executor.group=yarn
 yarn.nodemanager.log-dirs=<%=@mounts.map{ |d| "file:///disk/#{d}/yarn/logs" }.join(",")%>
 banned.users=hdfs,yarn,mapred,bin      
-min.user.id=1000
+min.user.id=100
```

Comment:
I don't believe we want to change this to 100 from 1000. Changing this to 100 will open doors for all other users and we don't want to do that.